content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -168,136 +196,142 @@ For example, if there are 1,253 pods on the cluster and the client wants to rece
 
+## Alternate representations of resources
 
-## Receiving resources as Tables
+### Receiving resources as tables
 
-`kubectl get` is a simple tabular representation of one or more instances of a particular resource type. In the past, clients were required to reproduce the tabular and describe output implemented in `kubectl` to perform simple lists of objects.
-A few limitations of that approach include non-trivial logic when dealing with certain objects. Additionally, types provided by API aggregation or third party resources are not known at compile time. This means that generic implementations had to be in place for types unrecognized by a client.
+By default, `kubectl get <kind>` prints a simple tabular representation of one or more instances of a particular resource type.
+It's useful for other API clients to be able to reproduce the same kind of formatted, tabular representation for a resource; this helps make user interfaces consistent.
 
-In order to avoid potential limitations as described above, clients may request the Table representation of objects, delegating specific details of printing to the server. The Kubernetes API implements standard HTTP content type negotiation: passing an `Accept` header containing a value of `application/json;as=Table;g=meta.k8s.io;v=v1beta1` with a `GET` call will request that the server return objects in the Table content type.
+Kubernetes API clients may request the Table representation of objects, delegating specific details of printing to the API server. The Kubernetes API uses HTTP content negotiation based on specialized media types; for example: passing an `Accept` header containing a value of `application/json;as=Table;g=meta.k8s.io;v=v1beta1` with a `GET` call will request that the server return a JSON document that serializes Kubernetes Table objects.
 
-For example, list all of the pods on a cluster in the Table format.
+For example, list all of the pods on a cluster, returning a Table serialized as JSON:
 
 ```console
-GET /api/v1/pods
+GET /api/v1/pods HTTP/1.1
 Accept: application/json;as=Table;g=meta.k8s.io;v=v1beta1
----
-200 OK
+…more headers…
+```
+```
+HTTP/1.1 200 OK
 Content-Type: application/json
 
 {
     "kind": "Table",
     "apiVersion": "meta.k8s.io/v1beta1",
-    ...
+    …
     "columnDefinitions": [
-        ...
+        …
     ]
 }
 ```
 
-For API resource types that do not have a custom Table definition on the server, a default Table response is returned by the server, consisting of the resource's `name` and `creationTimestamp` fields.
+For API resource types that do not have a custom Table definition known to the cluster, the API server prepares a default Table response that consists of the resource's `name` and `creationTimestamp` fields. For example:
```

Comment:
nit: cleanup: For API resource types that do not have a custom `Table` definition, the API server prepares ...